src/test/scala/uk/gov/hmrc/test/ui/cucumber/stepdefs/StepDefinitions.scala

```diff
@@ -361,7 +361,10 @@ class StepDefinitions
 
   Then("I will be navigated to the Check Your Answers page")(() => CheckYourAnswers.loadPage())
 
-  And("I check my answers and click on continue")(() => CheckYourAnswers.submitPage())
+  And("I check my answers and click on continue") { () =>
+    CheckYourAnswers.submitPage()
+    Thread.sleep(5000)
```

Comment:
I haven't seen this used elsewhere in the tests, what is it waiting for here?

Reply:
This particular page was failing to load on time on staging environment(worked in local without the wait) so had to add wait here. I think it was present earlier as well but I removed in previous PR thinking it's unnecessary. But now I know it's for staging

Reply:
Yeah sleeps aren't good in tests (they increase the time taken to run the suite). I wonder if there is a better way. Are we waiting for something to appear on the screen?

Reply:
Ideally we would do something like loop until the related text is on the screen. If the sleep was already existing in a previous version then do we want to spend time now to refactor?